engine/admin/volumes/volumes.md

```diff
@@ -0,0 +1,341 @@
+---
+description: Using volumes
+title: Use volumes
+keywords: storage, persistence, data persistence, volumes
+redirect_from:
+- engine/tutorials/dockervolumes/
+---
+
+Volumes are the preferred mechanism for persisting data generated by and used
+by Docker containers. While [bind mounts](bind-mounts.md) are dependent on the
+directory structure of the host machine, volumes are completely managed by
+Docker. Volumes have several advantages over bind mounts:
+
+- Volumes are easier to back up or migrate than bind mounts.
+- You can manage volumes using Docker CLI commands or the Docker API.
+- Volumes work on both Linux and Windows containers.
+- Volumes can be more safely shared among multiple containers.
+- Volume drivers allow you to store volumes on remote hosts or cloud providers,
+  to encrypt the contents of volumes, or to add other functionality.
+- A new volume's contents can be pre-populated by a container.
+
+In addition, volumes are often a better choice than persisting data in a
+container's writable layer, because using a volume does not increase the size of
+containers using it, and the volume's contents exist outside the lifecycle of a
+given container.
+
+If your container generates non-persistent state data, consider using a
+[tmpfs mount](tmpfs.md) to avoid storing the data anywhere permanently, and to
+increase the container's performance by avoiding writing into the container's
+writable layer.
+
+Volumes use `rprivate` bind propagation, and bind propagation is not
+configurable for volumes.
```

Comment:
If you need this you are already a Linux geek. I've already explained that it's an advanced option. I don't think I should spend more time on it.